src/main/resources/application.yaml

```diff
@@ -134,13 +137,15 @@ camunda:
       queue-capacity: 3
       wait-increase-factor: 2
       wait-time-in-millis: 15000
-    # see: https://docs.camunda.org/manual/7.20/user-guide/spring-boot-integration/configuration/#generic-properties 
+    # see: https://docs.camunda.org/manual/7.20/user-guide/spring-boot-integration/configuration/#generic-properties
     generic-properties:
       properties:
         telemetry-reporter-activate: false
         enable-script-compilation: true
         enable-script-engine-nashorn-compatibility: true
         enable-fetch-script-engine-from-process-application: false
+    metrics:
+      enabled: false
```

Comment:
Wonderful!
That has been ever so annoying.

Reply:
Luckily it broke when deployed in folio/snapshot. Turning it off solves both.